src/server-browser.js

```diff
@@ -0,0 +1,2 @@
+const KiteServer = {}
+export default KiteServer
```

Comment:
It might be useful to make this a `class` to make it not throw if a `KiteServer` wanted to be initialized in the browser.